CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.1]
```

Comment:
Is this a bug in that CloudEvents could not be compared, or a feature in providing a nice way to compare CloudEvents?

Reply:
```python
cloudevent1 == cloudevent2
```
would only test the memory address instead of an actual equality test because we forgot to override the __eq__ operator. I'd say both of the above are applicable :)